src/Modules/Reports/Commands.Reports/NewPowerBIReport.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+using Microsoft.PowerBI.Common.Client;
+using Microsoft.PowerBI.Common.Api.Reports;
+using Microsoft.PowerBI.Common.Api.Workspaces;
+using Microsoft.PowerBI.Common.Abstractions;
+
+namespace Microsoft.PowerBI.Commands.Reports
+{
+    [Cmdlet(CmdletVerb, CmdletName)]
+    [OutputType(typeof(Report))]
+    public class NewPowerBIReport : PowerBIClientCmdlet
+    {
+        public const string CmdletVerb = VerbsCommon.New;
```

Comment:
I think the CmdletVerb should be Add. @CodeCyclone, what do you think is best here?